restapi/api/reservation/reserve_endpoints.go

```diff
@@ -0,0 +1,138 @@
+package reservation
+
+import (
+	"math/rand"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/gin-gonic/gin"
+)
+
+var reservedDevicesMap = make(map[string]*reservedDevice)
+var reserveMutex sync.Mutex
+
+type reservedDevice struct {
+	Message           string `json:"message,omitempty"`
+	UDID              string `json:"udid,omitempty"`
+	ReservationID     string `json:"reservationID,omitempty"`
+	LastUsedTimestamp int64  `json:"lastUsed,omitempty"`
+}
+
+func CleanReservationsCRON() {
+	defer reserveMutex.Unlock()
+
+	// Every minute loop through the map of reserved devices and check if a reserved device last used timestamp was more than 5 minutes(300000 ms) ago
+	// If any, remove them from the map
+	for range time.Tick(time.Second * 60) {
+		reserveMutex.Lock()
+		for udid, reservedDevice := range reservedDevicesMap {
+			currentTimestamp := time.Now().UnixMilli()
+			diff := currentTimestamp - reservedDevice.LastUsedTimestamp
+
+			if diff > 300000 {
```

Comment:
nit: is something like time.Minute*5 possible here?
also would be good to have it in a const, I am planning to make this configurable later

Reply:
added an exported variable `ReservedDevicesTimeout` with default value of 5, can be accessed from main package to make it configurable